src/browser.js

```diff
@@ -1,40 +1,27 @@
-export function getBrowser() {
-  return typeof browser !== 'undefined' ? browser : chrome;
-}
+export const browserAPI = typeof browser !== "undefined" ? browser : chrome;
 
 export async function getCurrentTabInfo() {
-  const tabs = await getBrowser().tabs.query({active: true, currentWindow: true});
+  const tabs = await browserAPI.tabs.query({
+    active: true,
+    currentWindow: true,
+  });
   const tab = tabs && tabs[0];
 
   return {
     url: tab ? tab.url : "",
-    title: tab ? tab.title : ""
+    title: tab ? tab.title : "",
   };
 }
 
-function useChromeStorage() {
-  return typeof chrome !== "undefined" && !!chrome.storage;
-}
-
 export function getStorageItem(key) {
-  if (useChromeStorage()) {
-    const result = chrome.storage.local.get([key]);
-    return result.then(data => data[key])
-  } else {
-    return Promise.resolve(localStorage.getItem(key));
-  }
+  return browserAPI.storage.local.get([key]).then((data) => data[key]);
 }
 
 export function setStorageItem(key, value) {
-  if (useChromeStorage()) {
-    return chrome.storage.local.set({[key]: value});
-  } else {
-    localStorage.setItem(key, value);
-    return Promise.resolve();
-  }
+  return browserAPI.storage.local.set({ [key]: value });
```

Comment:
Storage is now well supported in both Chrome and FF https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/storage

Reply:
Maybe we can do a 1-time upgrade for those FF users? Something like reading from localStorage and writing those values to storage, then clearing local storage.